cores/esp8266/time.cpp

```diff
@@ -60,7 +61,7 @@ static void setServer(int id, const char* name_or_ip)
     }
 }
 
-
+extern "C++"
```

Comment:
Why are these `extern C++`?  Just because they're in an `ifdef _cpp` block in `Arduino.h`? 

I think, for sanity's sake, it makes sense to drop the `extern C++` on these functions and move the fcn prototype in Arduino.h out of the `ifdef _cpp` block (and making all these linkages C).   We can't go the other way because the other fcns here are part of POSIX C stuff.

What do you think?

Reply:
I agree with @earlephilhower. Is there a reason to label them like this?

Reply:
The reason is that `configTime` was "C" but I overloaded it with another one. So it needs to be "C++".
I removed this "C++" and moved them outside from `extern "C"`.